translations/translated/mudlet_en_US.ts

```diff
@@ -4,140 +4,90 @@
 <context>
     <name>GLWidget</name>
     <message numerus="yes">
-        <location filename="../../src/glwidget.cpp" line="383"/>
+        <location filename="../../src/glwidget.cpp" line="290"/>
         <source>You have a map loaded (%n room(s)), but Mudlet does not know where you are at the moment.</source>
         <translation>
-            <numerusform>You have a map loaded (one room), but Mudlet does not know where you are at the moment.</numerusform>
+            <numerusform>You have a map loaded (1 room), but Mudlet does not know where you are at the moment.</numerusform>
             <numerusform>You have a map loaded (%n rooms), but Mudlet does not know where you are at the moment.</numerusform>
         </translation>
     </message>
 </context>
 <context>
-    <name>T2DMap</name>
+    <name>MapInfoContributorManager</name>
```

Comment:
These next few changes are left over from the extension to the map info system - which was spun out to a new class. As such the `mudlet_en_US.ts` should have been updated then, but it wasn't until now. So this is wanted.